test-tools/dynamic-aot-debug/README.md

```diff
@@ -0,0 +1,84 @@
+# Dynamic AOT Module Debugging
+
+This guide outlines the process for debugging WAMR AOT module compiled modules using dynamic AOT debugging features. Follow the steps below to set up and run your debugging environment.
+
+## How to use
+
+### 1. Enable Debug Configuration
+
+To enable dynamic AOT debugging, set the following
+compile macro switch:
+
+```
+cmake -DWASM_ENABLE_AOT=1
+cmake -DWAMR_BUILD_DYNAMIC_AOT_DEBUG=1
+```
+
+### 2. Build AOT and Object Files
+
+#### 2.1 Build wamrc Compiler
+
+Ensure that wamrc is built with the WAMR_BUILD_DEBUG_AOT flag enabled. You can find the wamrc compiler [here](https://github.com/bytecodealliance/wasm-micro-runtime/tree/main/wamr-compiler)
```

Comment:
How about changing the last sentence to `
Please refer to the first two steps in [doc/source_debugging_aot.md](../../doc/source_debugging_aot.md).`?

Reply:
done